contrib/hyperscan/matching/input_matchers/source/matcher.cc

```diff
@@ -0,0 +1,107 @@
+#include "contrib/hyperscan/matching/input_matchers/source/matcher.h"
+
+#include <numeric>
+
+namespace Envoy {
+namespace Extensions {
+namespace Matching {
+namespace InputMatchers {
+namespace Hyperscan {
```

Comment:
nit: add blank line here (symmetry with below, consistency with most Envoy files)

Reply:
Thanks, I will change it.